tests/components/Nav.test.js

```diff
@@ -1,7 +1,10 @@
 import { render } from '@testing-library/react'
 import Nav from '@/components/Nav'
+import * as nextRouter from 'next/router'
 
 describe('<Nav />', () => {
+  nextRouter.useRouter = jest.fn() // eslint-disable-line
```

Comment:
Please move over to jest module mocks. This current set has potential to cause unexpected side effects when running tests. See example [here](https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/tests/setup-test-env.js#L10]
)